docs/concepts/registries.md

```diff
@@ -10,7 +10,7 @@ Every type of registrable object has its own registry. To see all registries sup
 Methods for Registering
 ------------------
 
-There are two proper ways to register objects: the `DeferredRegister` class, and the `RegistryEvent$Register` lifecycle event.
+There are three proper ways to register objects: the `DeferredRegister` class, the `RegistryEvent$Register` lifecycle event, as well as `Registry#register` for vanilla-only registries.
```

Comment:
This should really not be a combined statement. First, a list followed by a context means that all elements in the list are in relation to said context. Second, these two statements should be separated. Vanilla-only registries is a special case when no Forge wrapped registry is present. There are a few reasons why this may happen, but in general it should be its own section and not wrapped.